lib/membrane/rtp/twcc_receiver.ex

```diff
@@ -158,8 +153,8 @@ defmodule Membrane.RTP.TWCCReceiver do
     actions = [end_of_stream: pad]
 
     if Map.has_key?(ctx.pads, pad),
-      do: {{:ok, actions}, state},
-      else: {:ok, buffer_actions(actions, pad, state)}
+      do: {actions, state},
```

Comment:
you can move the content of that variable here
```suggestion
      do: {[end_of_stream: pad], state},
```

Reply:
`actions` variable is used two times, so removing it would cause repeating this same code

Reply:
You're right, I've missed that, sorry